docs/Usage.md

```diff
@@ -0,0 +1,216 @@
+  `--per-tilt-weighting` flag.
+* CTF data: a `.defocus` file from IMOD or `.txt` file to `--defocus-file`. The `.
+  txt` file 
+  should specify the defocus of each tilt in $\mu m$. You can also give a 
+  single defocus value (in $\mu m$). The CTF will also require input for 
+  `--voltage`, `--amplitude-contrast`, and `--spherical-abberation`.
+* Dose weighting: a `.txt` file to `--dose-accumulation` with the accumulated dose per 
+  tilt (assuming the same ordering as `.tlt`). Each line contains a single float 
+  specifying the accumulated dose in $e^{-}/\AA^{2}$. Dose weighting only works in 
+  combination with `--per-tilt-weighting`.
+
+_(As a side note, you can also only enable `--per-tilt-weighting` **without** dose accumulation and CTFs, or **with either** dose accumulation or CTFs.)_
+
+When enabling the CTF model here (with the defocus file), it is important that the template is not multiplied with a CTF before passing it to this script. The template only needs to be scaled to the correct pixel size and the contrast should be adjusted to match the contrast in the tomograms.
+
+Secondly, if the tomogram was CTF corrected, for example by using IMODs 
+strip-based CTF correction or NovaCTF. Its important to add the parameter 
+`--tomogram-ctf-model phase-flip` which modifies the template CTF to match the 
+tomograms CTF correction.
+
+#### Background corrections
+
+The software contains two background correction methods that might improve results: 
+`--spectral-whitening` or `--random-phase-correction` (from STOPGAP). In our 
+experience the random phase correction is most reliable, while spectral whitening 
+never seemed to clearly improve results.
+
+```python exec="on" result="ansi"
+import argparse
+code = ("""
+---8<--- "entry_points.py:match_template_usage"
+""")
+cleaned_lines = [line.lstrip() for line in code.splitlines()
+    if not any(keyword in line for keyword in ('action=', 'default=', 'type='))
+]
+cleaned_code = '\n'.join(cleaned_lines)
+exec(cleaned_code)
+print(parser.format_help())
+```
+
+## 3. Extracting particles
+
+Both scripts run on the job file created in `pytom_match_template.py` which contains details about correlation statistics and the output files. The job file always has the format `[TOMO_ID]_job.json`.
+
+**IMPORTANT** For both scripts the `[-r, --radius-px]` option needs to be considered carefully. The particle extraction will mask out spheres with this radius around each peak in the score volume and prevents selecting the same macromolecule twice. It is specified as an integer **number of pixels** (not Angstrom!) and ideally it should be the radius of the particle of interest. It can be found by dividing the particle radius by the pixel size, e.g. a ribosome (r = 290A / 2) in a 15A tomogram should gets a pixel radius of 9.6. As it needs to be an integer value and ribosomes are not perfect spheres, it is best to round it down to 9 pixels. 
```

Comment:
use angstrom sign
```suggestion
**IMPORTANT** For both scripts the `[-r, --radius-px]` option needs to be considered carefully. The particle extraction will mask out spheres with this radius around each peak in the score volume and prevents selecting the same macromolecule twice. It is specified as an integer **number of pixels** (not Angstrom!) and ideally it should be the radius of the particle of interest. It can be found by dividing the particle radius by the pixel size, e.g. a ribosome (r = 290Å / 2) in a 15Å tomogram should gets a pixel radius of 9.6. As it needs to be an integer value and ribosomes are not perfect spheres, it is best to round it down to 9 pixels. 
```